### DISCUSSION TITLE
Is it possible to customize the visible time window in the calendar view?

### DISCUSSION THREAD

**Discussion by mikedolx** [kimai/kimai#2607] [Questions & Answers] [answered]
Hi,

I'd like to know if it's possible to customize the calendar view to reduce shown time window. Currently the calendar view for a week and day shows the whole day from 0:00 to 24:00. Due to that I always have to scroll a bit to get to the "interesting" part of the calendar.
I'd like to only see time bookings between let's say 07:00-18:00. Is it possible to configure that somewhere?

Thanks,

BR,

Michael

**Comment by kevinpapst** [comment, marked as answer]
<img width="835" alt="Bildschirmfoto 2021-06-10 um 10 06 46" src="https://user-images.githubusercontent.com/533162/121488697-9b8e2a80-c9d3-11eb-9e4c-261f12420697.png">

**Reply by mikedolx** [reply]
Oh boy - that was a no brainer 👍  . Should have checked the admin settings before. Thanks!
